lib/cirro_io/client/reward.rb

```diff
@@ -0,0 +1,7 @@
+module CirroIO
+  module Client
+    class Reward < Base
```

Comment:
It looks like object type is tied to the URL here 🤔 The type is `bonus-payouts` in fact. Or we should rename it ASAP.